src/frontend/js/types/Joanie.ts

```diff
@@ -170,6 +170,13 @@ export interface Address {
   title: string;
 }
 
+// Wishlist
+export interface UserWishlistCourse {
+  id: string;
+  courseCode: Course['code'];
+  userId: number;
```

Comment:
Should be an `uuid` 🤔
```suggestion
  userId: string;
```